### PR TITLE
Updated Codeable integration

### PR DESCRIPTION
**Pull request by ahanzek**

### CODE REVIEW

**Comment by raleighleslie** [comment]
This would be awesome to see this Merged.

**Comment by IndrekV** [comment]
I haven't been able to test it as it seems my Codeable account is not functioning. I've contacted the Codeable team to solve it. Thank you for showing your support.

File: src/scripts/content/codeable.js [hunk 1]
```diff
@@ -2,16 +2,28 @@
...
+        third = document.querySelector('.sub-tasks .number');
+
+      first = (!!first ? first.textContent.trim() : "");
+      second = (!!second ? " · " + second.textContent.trim() : "");
+      third = (!!third ? " · Additional tasks: " + third.textContent.trim() : "");
+      return first + second + third;
+    },
+
+    project = function () {
+      return 'CODEABLE';
```
Comment (Phenome) [inline src/scripts/content/codeable.js]: This doesn't need to be a function. You can also just pass in a string.

**Comment by dooart** [comment]
@IndrekV Can you try contacting them again? I tried to sign up as a developer but there's an application process.

**Comment by IndrekV** [comment]
It seems my account has also been disabled. I'll write to them to see what's up. Last time I wrote to their support about a testing account and they promoted the account to 'contractor' status so I could test the integration.

**Comment by raleighleslie** [comment]
Hey! Thanks. I work in-house at Codeable and could help with testing account access for a limited time raleigh@codeable.io

I can convert your account if you can let me know when and the email for it?

Thanks I know our experts would love to use the toggl intergration again! -Raleigh

@IndrekV @dooart

**Comment by IndrekV** [comment]
@raleighleslie I'm using `indrek@toggl.com` for the testing account.

**Comment by bryanph** [comment]
@raleighleslie would be great if you could provide us with a permanent test account. The account we use for testing integrations is togglbutton@toggl.com.

I'll close this PR for now until we get some clarity on when this will be fixed (and we get a test account going). Instead I created a separate issue here: #1188. Please discuss further there.

**Comment by jonathanbossenger** [comment]
@bryanph were you folks able to test this with a view to merging it?

**Pull request closed without merging**